### PR TITLE
Memory container: fix issues with naturalKeyField storing

### PR DESCRIPTION
**Pull request by smoskalenko**

### CODE REVIEW

File: test/Persisted/Container/MemoryTest.php [hunk 1]
```diff
@@ -1,19 +1,21 @@
 <?php
 namespace Magomogo\Persisted\Container;
 
+use Magomogo\Persisted\AbstractProperties;
```
Comment (Magomogo) [inline test/Persisted/Container/MemoryTest.php]: Please review what namespaces are in use.

File: test/_classes/Magomogo/Persisted/Test/Affiliate/Cookie/Properties.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+class Properties extends AbstractProperties
+{
+    public $id;
+    public $lifeTime = 0;
+
+    public function naturalKeyFieldName()
+    {
+        return 'id';
+    }
+}
```
Comment (Magomogo) [inline test/_classes/Magomogo/Persisted/Test/Affiliate/Cookie/Properties.php]: code style: a new line is needed

Comment (smoskalenko) [reply test/_classes/Magomogo/Persisted/Test/Affiliate/Cookie/Properties.php]: @Magomogo both issues are fixed

**Comment by Magomogo** [comment]
Thanks. Patch version `v1.0.1` is issued.

**Pull request merged**